JavaScript/Issues_Bomb/README.md

```diff
@@ -0,0 +1,18 @@
+# Issues Bomb
+A script created to bomb multiple issues with Spaghetti Text into a selected repo (using Javascript)
+
+## Get Started
+- To Install Packages<br>
+```
+npm ci
+```
+
+- Run Code<br>
+```
+npm run bomb
+```
+
+## Screenshots
+<img src = "https://drive.google.com/u/0/uc?id=1u5VOO4qJ5SRkMF5faLAEAx93GDBgrh1Z&export=download"/>
```

Comment:
You can refer these Images by loading them on `Imgur` and then adding their Image URLs